cscope.py

```diff
@@ -243,10 +240,6 @@ def run_cscope(self, mode, word):
         output, erroroutput = proc.communicate()
         # print output
         # print erroroutput
-        try:
-            output = str(output, encoding="utf8")
-        except TypeError:
-            output = unicode(str(output), encoding="utf8")
 
```

Comment:
Why remove lines 246-249? I seem to recall we added this to handle a certain case and without it we'd hit an exception. Is it important to you that we remove this? Does it solve other problems or something? I'd rather leave this here unless there's a good reason to move it.